akeneo-design-system/src/components/Search/ResultCount.tsx

```diff
@@ -0,0 +1,28 @@
+import React, {ReactNode, Ref} from 'react';
+import styled from 'styled-components';
+import {getColor} from '../../theme';
+import {Override} from '../../shared';
+
+const ResultCountContainer = styled.div`
+  white-space: nowrap;
+  color: ${getColor('purple', 100)};
```

Comment:
```suggestion
  color: ${getColor('brand', 100)};
```
So that it's changed automatically on other Akeneo Products :)